backend/src/main/java/fk/prof/backend/util/ZookeeperUtil.java

```diff
@@ -0,0 +1,55 @@
+package fk.prof.backend.util;
+
+import io.vertx.core.Future;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.zookeeper.KeeperException;
+
+public class ZookeeperUtil {
+
+  public static byte[] readZNode(CuratorFramework curatorClient, String zNodePath)
+      throws Exception {
+    return curatorClient.getData().forPath(zNodePath);
+  }
+
+  public static void writeZNode(CuratorFramework curatorClient, String zNodePath, byte[] data, boolean create)
+      throws Exception {
+    if(create) {
+      curatorClient.create().forPath(zNodePath, data);
+    } else {
+      curatorClient.setData().forPath(zNodePath, data);
+    }
+  }
+
+  //TODO: Keeping this around in case required for policy CRUD. If not used there, remove
```

Comment:
Prefer killing code and then bringing it back if required. We have VCS, there is no reason to not kill it if it is unused as of now.